README.md

```diff
@@ -180,9 +180,11 @@ To pass arguments to `docstr-coverage`, add a `.docstr.yaml` config (which is au
 
 ```yaml
 - repo: https://github.com/HunterMcGushion/docstr_coverage
-  rev: <most recent docstr-coverage release or commit sha>
+  rev: v2.1.0 # most recent docstr-coverage release or commit sha
   hooks:
     - id: docstr-coverage
+      entry: bash -c 'docstr-coverage' -- # custom entrypoint to always prefer the paths from .docstr.yaml
```

Comment:
Might there be an other workaround for this issue? As you note in the PR description bash is not available on all systems.

Reply:
I wonder if pre-commit has an "official" solution to the problem. I'll look into it again 👍

Would you consider making it a part of hook if we find a better way around?

Reply:
Ok, I have good news: pre-commit provides a parameter that does just this: `pass_filenames: false`. 

It was interesting to learn that the pre-commit repo does not use this with its own mypy hook ([here](https://github.com/pre-commit/pre-commit/blob/2d4de2ef35fdb72d8af77c9907c42cfb588cb4e3/.pre-commit-config.yaml#L48-L51) and [here](https://github.com/pre-commit/mirrors-mypy/blob/master/.pre-commit-hooks.yaml)).

Judging from the discussion [in another issue](https://github.com/pre-commit/pre-commit/issues/1784) (and [this issue](https://github.com/pre-commit/pre-commit/issues/1519#issuecomment-646138780) too), it's not generally suggested because of poor performance. 

But given the nature of coverage checking tools, I think we should move the `pass_filenames` to `false` in the hook definition. @HunterMcGushion this can be a breaking change for anyone who already uses the hook, however I don't think it would affect too many people.

Reply:
Oh well, we shouldn't make it a default. This would break for people without the configuration file:

```
docstr-coverage..........................................................Failed
- hook id: docstr-coverage
- exit code: 1

No Python files found (use `--accept-empty` to exit with code 0 if you expect this case)
```

Reply:
I like explicit over implicit behavior, so I would be for the breaking change where we add `pass_filenames: false` 👍 . Performance is not an argument in my opinion, since as you're saying the nature of docstr coverage checking is that we always scan the entire repository. In the end not my call, we have to consult @HunterMcGushion for that.

Reply:
Having reviewed the original PR, let me just drop my 2 cents here:
I agree with @bjornconstructors - let's accept the breaking change (it has only been online for a few days and the example on our readme will show how to fix it). It definitely sounds like a better solution in the long run.

Reply:
Ok, I like that! I'll push the changes in a minute.

Maybe we could change the error message so that it suggests creating a config file?

```
No Python files found (use `--accept-empty` to exit with code 0 if you expect this case)
```

would change to something like

```
No Python files found. Use `--accept-empty` to exit with code 0 if you expect this case, or specify the paths you'd like to check via command line arguments or the config file.
```

However, not sure I've nailed it 😅

Reply:
Thank you, everyone, for working on developing the pre-commit hook feature. I apologize for my absence from the discussion, but thank you for continuing nonetheless. I love everything being added in this PR, but I am a bit concerned about pushing a breaking change. At the very least, we'll want to release this as 2.2.0, but I'd prefer to keep breaking changes as major releases, which would mean 3.0.0, but that seems a bit odd given the size of the change. Does anyone have any other ideas to avoid a breaking change?

Reply:
Also, just to be clear, I agree with @bjornconstructors that performance isn't an issue. I'm solely concerned with how best to release the breaking change

Reply:
I think releasing 2.2.0 would be more than enough. This is basically a fix for an otherwise broken behavior.